app/__init__.py

```diff
@@ -202,5 +204,19 @@ def update_sent_state(sender=None, body=None, **kwargs):
 scheduler.add_job(send_event_fee_notification_followup, 'cron', day=15)
 scheduler.start()
 
+
+@app.errorhandler(500)
+def internal_server_error(error):
+    if 'PROPOGATE_ERROR' in current_app.config:
```

Comment:
I don't think this check is needed. the config will always have `PROPOGATE_ERROR` right